docs/source/running.rst

```diff
@@ -181,6 +181,24 @@ is simple to run from the command, e.g:
 .. note::
    Uvicorn is as easy to install as ``pip install uvicorn``
 
+
+Django
+^^^^^^
+
+`Django`_ is a Python web framework that encourages rapid development and clean, pragmatic design.  Assuming
+a Django install/enabled application:
+
+.. code-block:: bash
+
+    python3 manage.py runserver
+
+In this section we create a sample Django project and use pygeoapi as an pluggable, embedded application.
+
+.. note::
```

Comment:
With the above comments we shouldn't need the note anymore

Reply:
note removed